tests/document/db_fields.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+import sys
+sys.path[0:0] = [""]
+import unittest
+
+from bson import SON
+from mongoengine import *
+
+__all__ = ("DbFieldParameterTest", )
+
+
+class DbFieldParameterTest(unittest.TestCase):
+
+    def setUp(self):
+        # testcase from https://github.com/MongoEngine/mongoengine/issues/904
```

Comment:
It would be better to add a docstring to the class above describing exactly what this set of tests ensures. Otherwise, we're forcing every dev to look up a (somewhat poorly worded) GitHub ticket in order to learn what this is about.